hack/patch_node.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+# This script can be used in order to create/remove/update k8s extended resources named prow/sriov
+# see https://kubernetes.io/docs/tasks/administer-cluster/extended-resource-node for more details.
+# The resource will be created node wise, so each sriov node should be configured according
+# the desired capacity (according how many PFs it has, and how many jobs are actually desired to run simultaneously).
+# After creation, the resource would appear as allocatable on node's yaml after around 20 seconds.
+# The following command can be used in order to check it was created successfully.
+# timeout 60s bash -c "until oc get node $NODE -oyaml | grep prow/sriov | grep $CAPACITY | wc -l | grep 2; do sleep 1; done"
+
+# Usage:
+# ./patch_node.sh <NODE_NAME> <CAPACITY>
+
+PROXY_PORT=9999
+
+function finish {
+  rc=$?
+  if jobs -p | xargs kill; then
+    echo -e "\nProxy deleted"
+  fi
+  exit "$rc"
+}
+
+function validate_parameters {
+  local node=$1
+  local capacity=$2
+  if ! oc get node "$node" &> /dev/null; then
```

Comment:
If we are planning to use this script in the new control plane and with the automated deployment tools we should use kubectl instead of  oc, is there any reason why you use oc here?

Reply:
Evgheni told me current CI cluster is using `oc`
maybe i will add auto detection according `which` in case needed ?
(tries to find oc, if not exists try to find kubectl, else fails)

Reply:
Yes, autodetection sounds good :+1: